src/dipole/transition_switcher.rs

```diff
@@ -0,0 +1,63 @@
+extern crate rayon;
+extern crate specs;
+use crate::atom::{Atom, AtomicTransition, Kind};
+use crate::destructor::ToBeDestroyed;
+use crate::dipole::atom::AtomicDipoleTransition;
+use crate::laser::cooling::CoolingLight;
+use crate::laser::dipole_beam::DipoleLight;
+use specs::{Entities, Join, LazyUpdate, Read, ReadStorage, System};
+extern crate nalgebra;
+
+/// System that globally deletes all `CoolingLight` entities that do not have
+/// a `DipoleLight`. This could be used to speed up the simulation after a completed MOT power-ramp
+/// and transition into a pure dipole trap.
+pub struct DisableMOTBeamsSystem;
```

Comment:
I think this should _not_ be included in the base library of atomecs.

We could think about making a set of tools/functionality for things like this. But this system is very user specific, and I think it should not be part of the base setup.

Reply:
I now agree - When I wrote this (a while back), I wasn't very certain about this whole API/Base AtomECS division thing. But yes, in the light of what we discussed it makes more sense to have that in our AION-specific repo